### PR TITLE
Output cutadapt stats and support custom read groups

### PR DESCRIPTION
**Pull request by mlaszloffy**

### CODE REVIEW

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

File: workflow-bwa-mem/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## BWA-MEM Workflow
 
-Version 1.1.1, SeqWare version 1.1.0
+Version 1.2, SeqWare version 2.0.2
```
Comment (morgantaschuk) [inline workflow-bwa-mem/README.md]: Niassa version 2.0.2 instead

File: workflow-bwa-mem/workflow/config/workflow.ini [hunk 2]
```diff
@@ -31,18 +31,27 @@ output_format=BAM
 do_trim=false
 # key=trim_mem_mb:type=integer:display=F
 trim_mem_mb=16384
+# key=cutadapt_log_mem_mb:type=integer:display=F
+cutadapt_log_mem_mb=16384
 # key=trim_min_quality:type=integer:display=F
-trim_min_quality=20
+trim_min_quality=0
```
Comment (morgantaschuk) [inline workflow-bwa-mem/workflow/config/workflow.ini]: Is this intentional?

**Pull request merged**